playbook/appherd/roles/apache/templates/httpd.conf.j2

```diff
@@ -0,0 +1,355 @@
+#
```

Comment:
This should be fine as long as the configuration matches what we're currently using save for the log formatting changes. Were you able to test this config at all? I would like to see that it is working properly before merging, so we might deploy to DEV or TEST to validate.

Reply:
I really need this to be deployed to DEV so we can see the impact on the Apache server and the splunk logging.